integration_tests/test_snap.py

```diff
@@ -81,7 +81,10 @@ def test_snap_default(self):
         self.assertThat(snap_file_path, FileExists())
 
     def test_cleanbuild(self):
-        self.skipTest("Fails to run correctly on travis.")
+        subprocess.check_call(['sudo', 'apt', 'install', '--yes', 'lxd'])
```

Comment:
We use apt-get elsewhere, so for consistency I'd prefer that. Also because apt causes warning spam (`WARNING: apt does not have a stable CLI interface. Use with caution in scripts.`).